### PR TITLE
update keymap of zergo keyboard

### PR DESCRIPTION
**Pull request by greenjack-z**
keymap changet to be up to date and in accordance with configurator json file

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/zergo/keymaps/default/keymap.c [hunk 1]
```diff
@@ -16,22 +16,35 @@
 
 #include QMK_KEYBOARD_H
 
+// Each layer gets a name for readability.
+#define l_main 0
+#define l_arrows 1
```
Comment (ridingqwerty) [inline keyboards/handwired/zergo/keymaps/default/keymap.c:21]:
```suggestion
enum zergo_layers {
    l_main,
    l_arrows,
};
```

I personally think the `enum` is better here, but IMO using `#define`s is not a dealbreaker for only two layers.  Up to you.

**Review by ridingqwerty — APPROVED** [review]
I made a small suggestion but don't feel obligated to commit it.

File: keyboards/handwired/zergo/keymaps/default/keymap.c (hunk 1 above)
Comment (greenjack-z) [reply keyboards/handwired/zergo/keymaps/default/keymap.c:21]: well, I'm not too deep in coding, so if that's preferable option I will add.

File: keyboards/handwired/zergo/config.h [hunk 2]
```diff
@@ -33,18 +33,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* key matrix pins */
 #define MATRIX_ROW_PINS { B1, D7, C3, D6, D5, D4 }
 #define MATRIX_COL_PINS { C7, C6, C5, C4, C2, C1, B7, D3, D2, B6, B5, B4, B3, B2 }
-#define UNUSED_PINS
+#define UNUSED_PINS { A0, A1, A2, A3, A4, A5, A6, A7, B0, C0, E0, E1, E4, E5, F0, F1, F2, F3, F4, F5, F6, F7 }
```
Comment (drashna) [inline keyboards/handwired/zergo/config.h:36]: Just a heads up, this value isn't actually used, and can be removed. 

Basically, it's only useful for reference.

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/zergo/config.h (hunk 2 above)
Comment (greenjack-z) [reply keyboards/handwired/zergo/config.h:36]: OK. I know

**Comment by drashna** [comment]
Thanks!

**Pull request merged**